<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	font-family:微软雅黑;
	font-size:15px;
	color: #1d649c;
}
#loadText{
	background-image: url(背景图.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>作者：一只安慕嘻</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>143</y>
     <width>991</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>歌曲列表</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>歌名</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="dictPath">
        <property name="text">
         <string>下载目录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startdownbutton">
        <property name="text">
         <string>开始下载</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopdownloadButton">
        <property name="text">
         <string>停止下载</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>下载信息</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="loadText"/>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>991</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>搜索栏</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="artistText">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>请输入你想下载的歌曲</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>245</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,1">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>获取评论页数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pageBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="ciyunButton">
      <property name="text">
       <string>生成词云</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="图标/tubiao1.qrc"/>
 </resources>
 <connections/>
</ui>
